### PR TITLE
feat: Add support to uncategorized articles

### PR DESCRIPTION
**Pull request by tejaswinichile**
https://linear.app/chatwoot/issue/CW-1306/add-the-support-for-creating-uncategorized-article

### CODE REVIEW

**Review by scmmishra — DISMISSED** [review]
The links to the articles seem broken in the dashboard UI and the Help Center UI

image

**Comment by tejaswinichile** [comment]
@nithindavid Yes, where did you mention the current routes? I need to update that too.

File: app/javascript/portal/components/SearchSuggestions.vue [hunk 1]
```diff
@@ -27,7 +27,7 @@
           :href="generateArticleUrl(article)"
           class="flex-auto truncate text-base font-medium leading-6 w-full hover:underline"
         >
-          {{ article.title }}
```
Comment (nithindavid) [inline app/javascript/portal/components/SearchSuggestions.vue:30]: Is this an intentional change?

Comment (tejaswinichile) [reply app/javascript/portal/components/SearchSuggestions.vue:30]: @nithindavid I was checking if this change affects anywhere in the UI, and it's not reflecting. As mentioned in the linear I am not able to change the routes here for public articles redirect.

**Review by nithindavid — CHANGES_REQUESTED** [review]
@tejaswinichile The article preview button on edit article header is not working, We might need to modify `app/javascript/dashboard/helper/portalHelper.js`

**Comment by tejaswinichile** [comment]
@nithindavid This has been resolved please have a look.

**Comment by nithindavid** [comment]
@tejaswinichile found this small bug. Is this intentional https://www.loom.com/share/16f2ffa8b48d4245b9c3e64db10186b4

**Comment by tejaswinichile** [comment]
@nithindavid Hey, I did not make changes to the private portal pages. I just made backend changes so that we can save the uncategorized article. This was one of the tasks to support the uncategorized article save whenever we make the UI changes accordingly.

The issue you mentioned was a feature always has a category for the article.

**Comment by 627721565** [comment]
您好，邮件已经收到，查阅后会尽快给您回复！

**Review by nithindavid — APPROVED** [review]

**Pull request merged**